src/nodes.ts

```diff
@@ -283,16 +283,19 @@ export class Conditional extends Node {
 }
 
 export class Program extends Node {
+  context: ParseContext;
+
   constructor(
     line: number,
     column: number,
     start: number,
     end: number,
     raw: string,
     readonly body: Block | null,
-    readonly context: ParseContext
+    context: ParseContext
   ) {
     super('Program', line, column, start, end, raw);
+    this.context = context;
```

Comment:
I don't know of one. I missed the assignment because I didn't see it in the diff. Thanks for setting this over the finish line!